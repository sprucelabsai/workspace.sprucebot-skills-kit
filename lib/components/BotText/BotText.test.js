'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _reactTestRenderer = require('react-test-renderer');

var _reactTestRenderer2 = _interopRequireDefault(_reactTestRenderer);

require('jest-styled-components');

var _BotText = require('./BotText');

var _BotText2 = _interopRequireDefault(_BotText);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

describe('BotText Component', function () {
	test('it renders', function () {
		var tree = _reactTestRenderer2.default.create(_react2.default.createElement(
			_BotText2.default,
			null,
			'Bottext children'
		)).toJSON();
		expect(tree).toMatchSnapshot();
	});
});