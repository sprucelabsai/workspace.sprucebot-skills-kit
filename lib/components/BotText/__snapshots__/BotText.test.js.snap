// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BotText Component it renders 1`] = `
<div
  className="bot__text "
>
  Bottext children
</div>
`;
